app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= render 'layouts/navbar' %> 
+
+<section class="flex-center column public-recipes">
+  <h2>Public Recipes</h2>
+  <ul class="max-width flex-center column">
+    <% @public_recipes.each do |recipe| %>
+      <li class="row">
+        <%= render partial:"public_recipes/public_recipe", locals: {recipe: recipe} %>
+      </li>
+    <% end %>
+  </ul>
+</section>
```

Comment:
- [ Required ] Good job adding the public_recipes page, however, I think when I click on a public recipe the application should take me to the recipe details page. Currently, when I click on any public recipe that I don't own, I get the error below. Please see the screenshot below for more details on this.


- [ Required ] Once you've implemented the change above :point_up: please ensure clicking on "Add ingredient" should lead to the form that allows the user to add a new ingredient, only if the user is the owner of the recipe, currently, I can't verify this since I can't access the page.

## User Journey
![Screenshot from 2023-04-28 12-05-44](https://user-images.githubusercontent.com/16392046/235107381-ccc00c7b-0202-4390-9440-9adce6a8074f.png)

## Error
![Screenshot from 2023-04-28 12-14-36](https://user-images.githubusercontent.com/16392046/235107771-2f7d2fe0-5b94-4856-904b-39aafb1735a3.png)

Reply:
Done

Reply:
Not resolved ✔️ 
Permission to view public recipes owned by other users is still denied.

![image](https://user-images.githubusercontent.com/38088818/235138882-702c7640-0874-4448-be3e-fe71666ed0c9.png)

Reply:
Oh, I understand now. Thank you 👏 !

Done